packages/lxd/test/query.coffee

```diff
@@ -15,3 +15,45 @@ describe 'lxc.query', ->
         path: '/1.0'
       $status.should.eql true
       data.api_version.should.eql '1.0'
+  
+  they 'with wait option', ({ssh}) ->
+    nikita
+      $ssh: ssh
+    , ->
+      {$status, data} = await @lxc.query
+        path: '/1.0'
+        wait: true
+      $status.should.eql true
+      data.api_version.should.eql '1.0'
+
+  they 'with get request option', ({ssh}) ->
+    nikita
+      $ssh: ssh
+    , ->
+      {$status, data} = await @lxc.query
+        path: '/1.0'
+        request: 'GET'
+      $status.should.eql true
+      data.api_version.should.eql '1.0'
+  
+  they 'stop a container with PUT request', ({ssh}) ->
+    nikita
+      $ssh: ssh
+    , ({registry}) ->
+      registry.register 'clean', ->
+        @lxc.delete 'nikita-query-1', force: true
+      await @clean()
+      await @lxc.init
+        image: "images:#{images.alpine}"
+        container: 'nikita-query-1'
+        start: true
+      {$status, data} = await @lxc.query
+        path: '/1.0/instances/nikita-query-1/state'
+        request: 'PUT'
+        data: '{"action": "stop", "force": true}'
+        wait: true
+      $status.should.eql true
+      {$status} = await @lxc.running
+        container: 'nikita-query-1'
+      $status.should.eql false
+      await @clean()
```

Comment:
```suggestion
      await @clean()

```

If you're using VSCode, there's a setting to force a final newline "Insert Final Newline" or in json:
```
"files.insertFinalNewline": true
```

Reply:
Thank you.